api/v1beta1/hetznercluster_types.go

```diff
@@ -48,7 +48,7 @@ type HetznerClusterSpec struct {
 	// +optional
 	ControlPlaneEndpoint *clusterv1.APIEndpoint `json:"controlPlaneEndpoint,omitempty"`
 
-	// ControlPlaneLoadBalancer is optional configuration for customizing control plane behavior. Naming convention is from upstream cluster-api project.
+	// ControlPlaneLoadBalancer is an optional configuration for customizing control plane behavior. The naming convention is from the upstream Cluster API project.
```

Comment:
// cc @janiskemper not a blocker but I think upstream only mandates `spec.controlPlaneEndpoint` 

Ref: https://main.cluster-api.sigs.k8s.io/developer/providers/cluster-infrastructure#infracluster-resources 

I think we can omit the part of naming convention given I was not able to find any upstream reference.

Reply:
```suggestion
	// ControlPlaneLoadBalancer is an optional configuration for customizing control plane behavior.
```

I agree, @Sayanta66 plz remove the second sentence